iron-component-page.html

```diff
@@ -317,7 +317,7 @@
            */
           transitive: {
             type: Boolean,
-            value: false
+            value: true
```

Comment:
This change seems like a breaking change. The catalog doesn't set it to false because the default is currently `false` https://github.com/Polymer/polymer-element-catalog/blob/b28c79add828d60bba26a05333e0220acceaea93/app/elements/pages/page-element.html#L82   

Looking at the description of this property, I wondered why it's good to change the default to true.